images/9.2.0 rev. 002893/windowsservercore/sitecore-xp-xc-base/Dockerfile

```diff
@@ -0,0 +1,49 @@
+# escape=`
```

Comment:
wrt to naming, as XC is always based on XP, wouldn't it be more logical to use `sitecore-xc-*`?

Reply:
Agree that we don't need xp-xc